### DISCUSSION TITLE
Multiprocessing

### DISCUSSION THREAD

**Discussion by ghost** [apache/airflow#25369] [General] [answered]
Hi, I am running the following code on my airflow and keep receiving the same error: "/usr/local/lib/python3.6/multiprocessing/process.py", line 103, in start
    'daemonic processes are not allowed to have children'
AssertionError: daemonic processes are not allowed to have children

the code
```
with concurrent.futures.ProcessPoolExecutor() as executor:
        for result in executor.map(scrap_url, make_url_list(f'{web_page}', f'{arg}')):
```
I read I should use from billiard import multiprocessing. Would you please be able to help on how to adapt the code to make it work on my airflow? It works locally...

**Comment by potiuk** [comment, marked as answer]
Just google billiard multiprocessing. you basically need to run `import billiard as multiprocessing' - but it could be other issue as well.
